src/_pytest/unittest.py

```diff
@@ -63,6 +62,13 @@ class UnitTestCase(Class):
     # to declare that our children do not support funcargs.
     nofuncargs = True
 
+    def newinstance(self, method_name: str = "runTest"):
```

Comment:
I understand we have been using `_` to separate words in new methods/functions? Anyway this is murky waters anyway, perhaps you wanted to keep consistency within this class.

Reply:
This overrides `Class.newinstance` so needs the same name